src/data/messages.json

```diff
@@ -4,188 +4,215 @@
     "sender":"Vladimir",
     "body":"why are you arguing with me",
     "timeStamp":"2018-05-29T22:49:06+00:00",
-    "liked": false
+    "liked": false,
+    "likeCount": 0
```

Comment:
Seems like you decided not to use the `likeCount` since you noticed that we could just use `liked` instead.